x-pack/plugins/maps/public/embeddable/map_embeddable_factory.ts

```diff
@@ -36,6 +69,7 @@ export class MapEmbeddableFactory implements EmbeddableFactoryDefinition {
   };
 
   async isEditable() {
+    await loadMapDependencies();
```

Comment:
Should we just do this once in a constructor?

Reply:
the imports load async, so there's no guarantee they have already loaded when being used. I renamed the function to improve clarity.